security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -323,18 +323,16 @@ func (sc *SecretManagerClient) addFileWatcher(file string, resourceName string)
 	if err := sc.tryAddFileWatcher(file, resourceName); err == nil {
 		return
 	}
-	// Retry file watcher as some times it might fail to add and we will miss change
+	// RetryWithContext file watcher as some times it might fail to add and we will miss change
```

Comment:
nit: this shouldn't change

Reply:
Seems not needed, as this is a comment for what the below does